ports/qtbase/cmake/qt_install_submodule.cmake

```
@@ -61,40 +62,44 @@ function(qt_cmake_configure)
         vcpkg_add_to_path(${PYTHON3_PATH})
     endif()
 
-    if(CMAKE_HOST_WIN32)
-        if(NOT ${PORT} MATCHES "qtbase")
-            list(APPEND _qarg_OPTIONS -DQT_SYNCQT:PATH="${CURRENT_HOST_INSTALLED_DIR}/tools/Qt6/bin/syncqt.pl")
-        endif()
-        set(PERL_OPTION -DHOST_PERL:PATH="${PERL}")
-    else()
-        if(NOT ${PORT} MATCHES "qtbase")
-            list(APPEND _qarg_OPTIONS -DQT_SYNCQT:PATH=${CURRENT_HOST_INSTALLED_DIR}/tools/Qt6/bin/syncqt.pl)
-        endif()
-        set(PERL_OPTION -DHOST_PERL:PATH=${PERL})
+    if(NOT ${PORT} MATCHES "qtbase")
+        list(APPEND _qarg_OPTIONS "-DQT_SYNCQT:PATH=${CURRENT_HOST_INSTALLED_DIR}/tools/Qt6/bin/syncqt.pl")
     endif()
+    set(PERL_OPTION "-DHOST_PERL:PATH=${PERL}")
 
     if(NOT _qarg_DISABLE_NINJA)
         set(NINJA_OPTION PREFER_NINJA)
     endif()

```

This also needs to be swapped:

```suggestion
    set(ninja_option "")
    if(_qarg_DISABLE_NINJA)
        set(ninja_option WINDOWS_USE_MSBUILD)
    endif()
```